src/main/kotlin/uk/gov/justice/digital/assessments/jpa/entities/refdata/AnswerDependencyEntity.kt

```diff
@@ -0,0 +1,12 @@
+package uk.gov.justice.digital.assessments.jpa.entities.refdata
+
+class AnswerDependencyEntity(
```

Comment:
My intention was to move this eventually in to the database, alongside the questions. Will rename if this is something we're looking to avoid and keep things out of the database

Reply:
Sorted - it's just a plain old Kotlin class now